cmd/scollector/collectors/snmp_ciscobgp.go

```diff
@@ -0,0 +1,190 @@
+package collectors
+
+import (
+	"fmt"
+	"io"
+	"strconv"
+	"strings"
+	"time"
+
+	"bosun.org/_third_party/github.com/mjibson/snmp"
+	"bosun.org/cmd/scollector/conf"
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+)
+
+const (
+	state               = ".1.3.6.1.4.1.9.9.187.1.2.5.1.3.1.4"
```

Comment:
these could also be declared in `c_snmp_ciscobgp`. They all also seem to share a prefix of `.1.3.6.1.4.1.9.9.187.1.2`. If you declare that separately and then concatenate in your add function, it may make them easier to look at. Or maybe they are easier to have all together for manual testing reasons. your call.